grid-proxy/pkg/client/grid_client_test.go

```diff
@@ -28,6 +30,67 @@ var (
 	NodeStatusExample = MarshalNodeStatus([]byte(NodeStatusExampleStr))
 )
 
+func nodesFilterValues() (types.NodeFilter, types.Limit, string) {
+	Up := "up"
```

Comment:
suggestion:
we can create a function with any value to be used directly in the filter instead of defining all the values:
```go
func ref[T any](v T) *T {
	return &v
}
```